recipes/alignlib-lite/meta.yaml

```diff
@@ -16,19 +18,18 @@ requirements:
     - llvm # [osx]
     - python
     - setuptools
-    - boost
+    - boost {{ CONDA_BOOST }}*
```

Comment:
Seems like alignlib is linking statically against boost. In such a case, we don't need the pinning. I will update the linter such that it ignores such cases.

Reply:
xref: bioconda/bioconda-utils#191

Reply:
How would you detect such cases via the linter?
Are you positive about the static linking (haven't checked myself)? So, should I just add the appropriate `skip-lints` for this recipe (with a comment that it links statically)?

Reply:
@johanneskoester / https://github.com/bioconda/bioconda-utils/pull/191:
> If the dependency does not occur in run, we don't need to pin it at all because it is statically linked.

Ok, got it! I'll wait for https://github.com/bioconda/bioconda-utils/pull/191 to be merged an then update this PR by removing the pinning.

Reply:
It is merged.